lab1.js

```diff
@@ -134,6 +150,15 @@ var tooHungryDay;
 */
 
 // your code goes here
+var averageMeals = 0;
+tooHungryDay = 0;
+for (var j = 0; j < mealsPerDay.length; j++) {
+  averageMeals += mealsPerDay[j];
+  console.log('Average meals since the caretaker started: ' + averageMeals / (j + 1));
+  if ((averageMeals / (j + 1)) < 4 && tooHungryDay === 0) {
```

Comment:
I like that you're checking to see if tooHungryDay has a value. An alternate method would be to use break to break the loop once you've found a tooHungryDay and assigned it to the tooHungryDay variable.